src/tools/tidy/src/features.rs

```diff
@@ -344,7 +344,7 @@ fn get_and_check_lib_features(base_src_path: &Path,
                 Ok((name, f)) => {
                     let mut check_features = |f: &Feature, list: &Features, display: &str| {
                         if let Some(ref s) = list.get(name) {
-                            if f.tracking_issue != s.tracking_issue {
+                            if f.tracking_issue != s.tracking_issue && f.level != Status::Stable {
```

Comment:
To confirm, was this change intended to be included here?

Reply:
Yes, the libcore change doesn't pass tidy without it.
Tidy expected a tracking issue in a `#[stable(...)]` attribute to coincide with something, but the problem is that `#[stable(...)]` doesn't support issue numbers - `#[stable(..., issue = "...")]` is not a valid attribute.

Reply:
👍